packages/dx-react-grid/docs/reference/table-column-resizing.md

```diff
@@ -34,6 +34,7 @@ maxColumnWidth? | number | `Infinity` | Specifies a column's maximum width.
 defaultColumnWidths? | Array&lt;[TableColumnWidthInfo](#tablecolumnwidthinfo)&gt; | [] | Specifies initial column widths in uncontrolled mode.
 onColumnWidthsChange? | (nextColumnWidths: Array&lt;[TableColumnWidthInfo](#tablecolumnwidthinfo)&gt;) => void | | Handles column width changes.
 columnExtensions? | Array&lt;[TableColumnResizing.ColumnExtension](#tablecolumnresizingcolumnextension)&gt; | [] | Additional column properties that the plugin can handle.
+columnResizingMode? | string | 'widget' | Specifies resizing mode. May be defined as 'widget' or 'nextColumn'. In 'widget' resizing mode not allow to use 'auto' or '%' sizes.
```

Comment:
I prefer to use this character ` instead of '. It highlights text in the site

Reply:
Can we change the name to `resizingMode` because this plugin also has the `column` word in its name?